From 74d8831e9008efbb344f781adf7803ebe1d25a78 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms@verge.net.au>
Date: Tue, 12 Feb 2013 10:45:50 +0900
Subject: CLEANUP: dumpstats: Make cli_release_handler() static

Make cli_release_handler() static, it is only used
inside dumpstats.c

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 src/dumpstats.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/dumpstats.c b/src/dumpstats.c
index 94e94f2..5954e8c 100644
--- a/src/dumpstats.c
+++ b/src/dumpstats.c
@@ -3992,7 +3992,7 @@ static int stats_dump_sess_to_buffer(struct stream_interface *si)
  * external abort, we won't call the i/o handler anymore so we may need to
  * remove back references to the session currently being dumped.
  */
-void cli_release_handler(struct stream_interface *si)
+static void cli_release_handler(struct stream_interface *si)
 {
 	if (si->applet.st0 == STAT_CLI_O_SESS && si->conn->xprt_st == STAT_ST_LIST) {
 		if (!LIST_ISEMPTY(&si->applet.ctx.sess.bref.users))
-- 
1.7.1

